### PR TITLE
Update multiple consumers info

### PR DESCRIPTION
**Pull request by vzabaznov**
Update information that multiple consumers should be used only with RabbitMQ

## Purpose of this pull request

This pull request (PR) ...

## Affected DevDocs pages

-  ...

## Links to Magento source code

-  ...

### CODE REVIEW

File: src/guides/v2.4/config-guide/mq/manage-message-queues.md [hunk 1]
```diff
@@ -66,6 +66,9 @@ Edit the `/app/etc/env.php` file to configure the cron job `consumers_runner`.
 *  `consumers` - An array of strings specifying which consumer(s) to run. An empty array runs *all* consumers.
 *  `multiple_processes` - An array of key-value pairs specifying which consumer to run in how many processes.
 
+   {:.bs-callout-info}
+   We do not recommned to run multiple consumers on Mysql operated queue. See [`how to change queue from Mysql to Rabbit`]({{ page.baseurl }}/guides/v2.3/extension-dev-guide/message-queues/message-queues.html#change-message-queue-from-mysql-to-amqp)
```
Comment (BarnyShergold) [inline src/guides/v2.4/config-guide/mq/manage-message-queues.md:70]:
```suggestion
   We do not recommend running multiple consumers on a Mysql operated queue. See [`how to change queue from Mysql to Rabbit`]({{ page.baseurl }}/guides/v2.3/extension-dev-guide/message-queues/message-queues.html#change-message-queue-from-mysql-to-amqp)
```

**Review by BarnyShergold — CHANGES_REQUESTED** [review]
Please also add this to the 2.3 version of the file - this can be done on this PR.
Please complete the PR description as required in the WIKI

**Comment by vzabaznov** [comment]
this functionality is not tested and might not work for 2.3.x

**Review by dobooth — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import pr to magento-devdocs/devdocs

**Comment by m2-github-services** [comment]
@dobooth the Pull Request is successfully imported.

**Pull request merged**